modules/programs/fabric-ai.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+let
+  inherit (lib)
+    mkIf
+    mkEnableOption
+    mkPackageOption
+    ;
+
+  inherit (lib.hm.shell)
+    mkBashIntegrationOption
+    mkZshIntegrationOption
+    ;
+
+  cfg = config.programs.fabric-ai;
+in
+{
+  meta.maintainers = with lib.hm.maintainers; [ aguirre-matteo ];
+  options.programs.fabric-ai = {
+    enable = mkEnableOption "Fabric AI";
+    package = mkPackageOption pkgs "fabric-ai" { nullable = true; };
+    enablePatternsAliases = mkEnableOption "aliases for all Fabric's patterns";
+    enableYtAlias = mkEnableOption "Fabric's `yt` alias" // {
+      default = true;
+    };
+    enableBashIntegration = mkBashIntegrationOption { inherit config; };
+    enableZshIntegration = mkZshIntegrationOption { inherit config; };
+  };
+
+  config =
+    let
+      posixShellCode = ''
+        ${lib.optionalString cfg.enablePatternsAliases ''
+          for pattern_file in $HOME/.config/fabric/patterns/*; do
```

Comment:
Is this always the same location for https://github.com/justfoolingaround/animdl/blob/master/animdl/core/config/__init__.py#L44